data/charts/istiocoredns/values.yaml

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
is all of this a copy of the existing charts?  I think it would be more appropriate to copy by reference rather than value :)

Reply:
yes, see comment above.